packages/cashier/src/Stores/general-store.js

```diff
@@ -250,13 +250,6 @@ export default class GeneralStore extends BaseStore {
                     }
                 }
             );
-
-            reaction(
-                () => currency,
-                () => {
-                    withdraw.setIsWithdrawConfirmed(false);
-                }
-            );
         }
```

Comment:
It's not needed (reaction) since the functionality is handled in line 113 of crypto-withdraw-receipt file, so we can remove this part (c) @Bahar-FS